### PR TITLE
Bug: TS types are missing for document.createElement default ESL tag names #332

### PR DESCRIPTION
**Pull request by NastaLeo**
global declaration of elements types

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "types": "modules/all.d.ts",
   "sideEffects": [
     "modules/lib.js",
```
Comment (ala-n) [inline package.json:32]: modules/namespace.js

**Review by ala-n — APPROVED** [review]

**Review by Sisha3342 — APPROVED** [review]

**Comment by ala-n** [comment]
Sorry, will use this pr to resolve typings issues.
The typing doesn't seem to work correctly with the strategy we choose, so update with distributed typings.
Also, there were some lint fixes, and namespace types rework.

Team, please be aware of the new type of placement.

**Pull request merged**